spark/src/test/scala/org/apache/mahout/sparkbindings/test/DistributedSparkSuite.scala

```diff
@@ -45,6 +45,7 @@ trait DistributedSparkSuite extends DistributedMahoutSuite with LoggerConfigurat
           .set("spark.akka.frameSize", "30")
           .set("spark.default.parallelism", "10")
           .set("spark.executor.memory", "2G")
+          .set("spark.kryo.registrationRequired", "true")
```

Comment:
Its been removed, the recent github changes for code review retain the original lines that were commented; confused me too.